munit/shared/src/main/scala/munit/Assertions.scala

```diff
@@ -90,7 +90,8 @@ trait Assertions {
   }
 
   def intercept[T <: Throwable](
```

Comment:
Thank you for this contribution! Is this the same signature as in scalatest? I feel like it would be better to have an interceptMessage assertion and leave intercept unchanged. What do you think?

Reply:
No worries!  No, scalatest has a different API, I think.  Most frameworks have some way to validate an Exceptions message, as this is very useful.  I added interceptMessage() as suggested because the API is cleaner.  Now you can say:
```scala
interceptMessage[MyException]("boom"){ 
  // scary code here
}
```